agenta-web/src/components/Evaluations/Evaluations.tsx

```diff
@@ -87,6 +78,31 @@ export default function Evaluations() {
         fetchData()
     }, [appName])
 
+    useEffect(() => {
+        if (variants.length > 0) {
+            const fetchAndSetSchema = async () => {
+                // Create a new object to accumulate the results for each variant
+                const newVariantsInputs: Record<string, string[]> = {}
+
+                for (let variant of variants) {
```

Comment:
instead of fetching each variant's params one by one, we can optimize by using `Promise.all`.

Reply:
fixed